pkg/noobaaaccount/reconciler.go

```diff
@@ -305,7 +307,20 @@ func (r *Reconciler) ReconcileDeletion() error {
 			return fmt.Errorf("failed to delete account %q. got error: %v", r.NooBaaAccount.Name, err)
 		}
 	} else {
-		r.Logger.Infof("✅ Successfully deleted account %q", r.NooBaaAccount.Name)
+		annotationValue, exists := util.GetAnnotationValue(r.NooBaa.GetAnnotations(), "remote-operator")
+		if exists {
+			if strings.ToLower(annotationValue) != strconv.FormatBool(false) {
+				nbremoteAuthToken := &corev1.Secret{}
+				nbremoteAuthToken.Name = r.NooBaaAccount.Name
+				nbremoteAuthToken.Namespace = r.NooBaaAccount.Namespace
+				err = r.Client.Delete(r.Ctx, r.Secret)
```

Comment:
If I'm not mistaken, R.Secret is deleted automtically, since it's owned by the noobaaAccount CR. I don't think that anything is changed in this PR with respect to deletion. Can you please verify it?